src/main/kotlin/nl/info/zac/flowable/bpmn/ZaaktypeBpmnProcessDefinitionService.kt

```diff
@@ -52,4 +52,14 @@ class ZaaktypeBpmnProcessDefinitionService @Inject constructor(
                 entityManager.createQuery(query).resultList
             }
         }
+
+    fun findByProductAanvraagType(productAanvraagType: String): ZaaktypeBpmnProcessDefinition? =
+        entityManager.criteriaBuilder.let { criteriaBuilder ->
+            criteriaBuilder.createQuery(ZaaktypeBpmnProcessDefinition::class.java).let { query ->
+                query.from(ZaaktypeBpmnProcessDefinition::class.java).let {
+                    query.where(criteriaBuilder.equal(it.get<String>("productaanvraagtype"), productAanvraagType))
```

Comment:
Minor thing but I think it's better to make a constant for `productaanvraagtype` in the `ZaaktypeBpmnProcessDefinition` class and use that here. I'll quickly do this on this PR.